crds/virtualmachinemacaddresses.yaml

```
@@ -0,0 +1,159 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: virtualmachinemacaddresses.virtualization.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: virtualization
+spec:
+  group: virtualization.deckhouse.io
+  scope: Namespaced
+  names:
+    categories:
+      - all
+      - virtualization
+    plural: virtualmachinemacaddresses
+    singular: virtualmachinemacaddress
+    kind: VirtualMachineMACAddress
+    shortNames:
+      - vmmac
+      - vmmacs
+  versions:
+    - name: v1alpha2
+      schema:
+        openAPIV3Schema:
+          description: |
+            The resource that defines MAC address for virtual machine.
+          properties:
+            apiVersion:
+              type: string
+            kind:
+              type: string
+            metadata:
+              type: object
+            spec:
+              description: |
+                The desired state of `VirtualMachineMACAddress`.
+              properties:
+                address:
+                  description: |
+                    The requested MAC address that should be assigned to the virtual machine.
```

Field is not required, description should say something about empty field:

```suggestion
                    The requested MAC address that should be assigned to the virtual machine. Random address is assigned if empty.
```

Also, change doc-ru